readme.md

```diff
@@ -44,79 +47,54 @@ If you can verify that this solved your issue,
 try increasing this number until you find a happy middle ground. For reference, the default number
 for incremental builds is `256`, and for non-incremental builds `16`.
 
-You also cannot set `linker = "rust-lld.exe"`, as subsecond currently crashes when `linker` is set.
-
 </details>
 
-<details>
-<summary>
-macOS
-</summary>
+#### MacOS
+
 
 You're in luck! Everything should work out of the box if you use the default system linker.
 
-</details>
+
+#### Linux
+
+Prerequisites: `clang` and either `lld` or `mold`
 
 <details>
 <summary>
-Linux
+Minimal config
 </summary>
 
-Execute the following:
-```sh
-readlink -f "$(which cc)"
-```
-If this points to `clang`, you're good. Otherwise, we'll need to symlink it.
-Read the path returned by the following command:
-```sh
-which cc
-```
-and `cd` into it. For example,
-
-```sh
-$ which cc
-/usr/bin/cc
-$ cd /usr/bin
-```
-
-Assuming you have `clang` installed, run the following commands:
-
-```sh
-mv cc cc-real
-ln -s "$(which clang)" cc
-```
-Note that the above commands may require `sudo`.
-
-Now everything should work. If not, install `lld` on your system and add the following to your `~/.cargo/config.toml`:
-
+Create or edit `./.cargo/config.toml`, with this minimal config
```

Comment:
```suggestion
Create or edit `~/.cargo/config.toml`, with this minimal config
```
that's more future proof for when the user want to use hot-patching in other repos

Reply:
I'll admit, changing `~/` to `./` here was an accident, i just re-wrote that without looking (i rewrote it a few times over the course of the day)

but if I'd have noticed it before, I'd have requested it changed: recommending global config which can affect far more than just bevy projects might seem like a good idea, but it will only lead to frustration when someone wants to work on a different project and gets odd errors due to the global config.

I'd far prefer the approach of integrating a config in templates, perhaps behind a setup question.

Reply:
feel free to accept whichever you prefer, i have "allow edits by maintainers" turned on, its 1:30am and i'm going to bed